natgeo/mortar
# [SYTO-1178] Fixed color zoom-in zoom-out icons

To test:

- Go to Icons menu
- Click on "Inverse fill color"
- Verify zoom-in and zoom-out icons have a white fill.
